readme.md

```diff
@@ -93,6 +93,7 @@ Full product comparison:
     <tr><td><b>Incognito mode detection</b><br/><i>works in all modern browsers - see our full list of <a href="https://dev.fingerprint.com/docs/browser-support/" target="_blank">browsers supported</a></i></td><td align="center">–</td><td align="center">✓</td></tr>
     <tr><td><b>Server-side accuracy increase</b><br/><i>based on additional server-side signals, such as TLS crypto support, ipv4/v6 data and others</i></td><td align="center">–</td><td align="center">✓</td></tr>
     <tr><td><b>Query API & realtime Webhooks</b><br/><i>build flexible workflows</i></td><td align="center">–</td><td align="center">✓</td></tr>
+    <tr><td><b>Ability to evade ad blockers</b></td><td align="center"><a href="docs/evade_ad_blockers.md" target="_blank">manual</a></td><td align="center">automatic (via custom domains)</td></tr>
```

Comment:
I don't think the comparison table is a good place to put this link because Pro version requires more effort to evade ad blockers

Reply:
In fact, the comparison table is a good place, but I don't agree that evasion in Pro version is automatic

Reply:
What do you suggest? Something like `out of the box (subdomain setup)`, `supported (subdomain setup)`,  or just `via subdomain setup`?

Reply:
Maybe?

| Open | Pro |
|--------|--------|
| supported (NPM, own hosting) | supported (NPM + subdomain) |

Both cells can include links BTW

Reply:
I don't think it's fair to say `supported` for both. There are significant pros for Pro and the `supported` label for both of them might imply a false feeling of equality (which is not true IMO). I mean, for open-source, this `supported` term means you need to implement it by yourself, and additionally, you have to give up automatic updates of the fingerprinting script because you need to use the static NPM package. On the contrary, this doesn't apply to Pro.

At first, I wanted to add also a link for the subdomain setup, nevertheless, I decided not to since we don't link Pro's features from the feature matrix at all.

Reply:
Something similar is proposed in the PR.

Reply:
Yes, you are right @Finesse with both (the NPM approach and Cloudflare integration capabilities). I was a little bit confused with the loader capabilities and its implications.

Reply:
Both "manual + automatic" and "manual + manual" are ok for me, because the table is not required to be 100% accurate. Please merge the PR with the variant that you like.